js/modules/k6/http/request.go

```diff
@@ -437,21 +437,25 @@ func (c *Client) prepareBatchObject(requests map[string]interface{}) (
 
 // Batch makes multiple simultaneous HTTP requests. The provideds reqsV should be an array of request
 // objects. Batch returns an array of responses and/or error
-func (c *Client) Batch(reqsV goja.Value) (interface{}, error) {
+func (c *Client) Batch(reqsValues ...goja.Value) (interface{}, error) {
 	state := c.moduleInstance.vu.State()
 	if state == nil {
 		return nil, ErrBatchForbiddenInInitContext
 	}
 
+	if len(reqsValues) == 0 {
+		return nil, fmt.Errorf("no argument was provided to http.batch()")
+	} else if len(reqsValues) > 1 { // Validation for http.batch(["https://validurl"],["https://validurl"])
```

Comment:
```suggestion
	} else if len(reqsValues) > 1 {
```
The message is already clear and we have also a specific test so we don't need this comment.

Reply:
Done